README.md

```diff
@@ -403,9 +403,9 @@ flavor: |
 
 * `latest=<auto|true|false>`: Handle [latest tag](#latest-tag) (default `auto`)
 * `prefix=<string>,onlatest=<true|false>`: A global prefix for each generated
-  tag and optionally for `latest`
+  tag. If `onlatest` (default `false) is set to `true`, also for the [latest tag](#latest-tag).
 * `suffix=<string>,onlatest=<true|false>`: A global suffix for each generated
-  tag and optionally for `latest`
+  tag. If `onlatest` (default `false) is set to `true`, also for the [latest tag](#latest-tag).
```

Comment:
```suggestion
  tag.

The `prefix` and `suffix` flavor settings can set `onlatest=true` (default: `false`) to additionally apply the affix to the [`latest` tag](#latest-tag) (_when it is added via the `flavor.latest` condition_).
```

I think we can avoid repeating that tip by shifting it down to appear after the list? This is a bit more wordier, but keeps the main descriptions terse.

It may possibly be worth linking [to this comment](https://github.com/docker/metadata-action/issues/567#issuecomment-3579068205) for more info, which I think is done elsewhere in the README for topics that might help users understand better when troubleshooting (_getting tripped up by the `latest` tag generation behaviour is a common one AFAIK_).